.github/workflows/pr_validation.yml

```diff
@@ -23,10 +23,10 @@ jobs:
             tests:
               - 'tests/**'
 
-      - name: Set up Python 3.11
+      - name: Set up Python 3.10
```

Comment:
docs.yml is python version 3.11, any specific reason why this is version 3.10?

Reply:
yes, ubuntu 22.04.4 LTS comes with python 3.10, not 3.11

(also: I got tempted into upgrading that on my system in an orthodox way, only to end up not being able to boot, and requiring  an os reinstall ...  other linux colleagues are on that same release, so...) 

Further down the road it might be even smart to investigate checking if we can also support even older releases, to ensure people on older releases can also use this stuff - even if the kgap-docker approach also helps in shielding that kind of issues... 

Hope we can make the doc run on 3.10 too?